10.9-libcxx/stable/main/finkinfo/devel/bison.info

```diff
@@ -40,15 +41,18 @@ CompileScript: <<
 InfoTest: <<
   TestDepends: ghostscript (>= 9.25)
   # rely on java/javac instead of gij for java testsuite
```

Comment:
I generally leave these sorts of notes (and likewise the `export JAVA` and `export JAVAC` setting) in case someone does want to play with things we disable in the public package.